app/controllers/projects_controller.rb

```diff
@@ -233,7 +233,7 @@ def set_story_flow
   end
 
   def allowed_params
-    params.fetch(:project,{}).permit(:name, :point_scale, :default_velocity, :start_date, :iteration_start_day, :iteration_length, :import, :archived, :disallow_join)
+    params.fetch(:project,{}).permit(:name, :point_scale, :default_velocity, :tag_group_id, :start_date, :iteration_start_day, :iteration_length, :import, :archived, :disallow_join)
```

Comment:
Please, break this in multiple lines. Line too long here.

Also, use `require(:project)` instead `fetch`.